src/components/auth/SignUp.js

```diff
@@ -30,158 +30,131 @@ function SignUp() {
-                  <li>Video Tutorials</li>
-                  <li>HarperDB Migrator</li>
-                </ul>
+  return (
+    <div className="login-form">
+      {formState.submitted ? (
+        <Loader header="creating your account" spinner relative />
+      ) : (
+        <>
+          <Form>
+            <h2 className="mb-2 instructions">Sign Up</h2>
+            <span className="mb-2 login-nav-link error d-inline-block">{formState.error}</span>
+            <Label className="mb-3 d-block">
+              <span className="mb-2 d-inline-block">First Name</span>
+              <Input
+                id="firstname"
+                name="fname"
+                required
+                autoComplete="given-name"
+                type="text"
+                title="first name"
+                placeholder="Jane"
+                value={formData.firstname || ''}
+                disabled={formState.submitted}
+                onChange={(e) => setFormData({ ...formData, firstname: e.target.value })}
+              />
+            </Label>
+            <Label className="mb-3 d-block">
+              <span className="mb-2 d-inline-block">Last Name</span>
+              <Input
+                id="lastname"
+                name="lname"
+                required
+                autoComplete="family-name"
+                type="text"
+                title="last name"
+                placeholder="Doe"
+                value={formData.lastname || ''}
+                disabled={formState.submitted}
+                onChange={(e) => setFormData({ ...formData, lastname: e.target.value })}
+              />
+            </Label>
+            <Label className="mb-3 d-block">
+              <span className="mb-2 d-inline-block">Email</span>
+              <Input
+                id="email"
+                autoComplete="email"
+                name="email"
+                required
+                className="mb-2"
+                type="text"
+                title="email"
+                placeholder="jane.doe@harperdb.io"
+                value={formData.email || ''}
+                disabled={formState.submitted}
+                onChange={(e) => setFormData({ ...formData, email: e.target.value.toLowerCase() })}
+              />
+            </Label>
+            <Label className="mb-3 d-block">
+              <Row>
+                <span className="mb-2 d-inline-block">Subdomain</span>
+                <Col className="subdomain-form">
+                  <Input
+                    id="subdomain"
+                    name="subdomain"
+                    required
+                    className=""
+                    type="text"
+                    title="subdomain"
+                    placeholder="janedev"
+                    value={formData.subdomain || ''}
+                    disabled={formState.submitted}
+                    onChange={(e) => setFormData({ ...formData, subdomain: e.target.value.substring(0, 15).toLowerCase() })}
```

Comment:
Where does the 16 character limit come from here? How is the UX around this? What if I enter more than 16 characters?

Reply:
I'll have to double check with @DavidCockerill  but this PR I put in week 1 may be related 🤔 
https://github.com/HarperDB/hdbms/pull/404